lib/fs.js

```diff
@@ -1617,8 +1617,15 @@ function ReadStream(path, options) {
   if (!(this instanceof ReadStream))
     return new ReadStream(path, options);
 
+  if (options === undefined)
```

Comment:
latest node and io.js accept `null` on fs.createReadStream.
current pull request does not change the behavior on fs.createReadStream, accepts `string` and throws proper exception.

if we don't accept `null`, this api may break the compatibility.